doc/source/build-actions/index.rst

```diff
@@ -23,7 +23,7 @@ Here is a code sample for using this action:
       runs-on: ubuntu-latest
       steps:
         - name: "Build library source and wheel artifacts"
-          uses: pyansys/actions/build-library@main
+          uses: pyansys/actions/build-library@v1
```

Comment:
Tags in development branch point to main. In the `release/0.1` they point to `v1`. No need to fix these in here.